ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -449,78 +532,189 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         </div>
         }
         <WorkspaceEditSection header='Billing Account' subHeader='National Institutes of Health'
-            tooltip={toolTipText.billingAccount}
-            text='To fulfill program requirements, All of Us requests the following information
-            for each workspace.'/>
-        <WorkspaceEditSection header='Describe your research purpose'
-            tooltip={toolTipText.researchPurpose}
-            text={['Please include the ', <strong key='question'>research question</strong>,
-              '\, the ', <strong key='plan'> plan for use of the data </strong>,
-              'to answer the research question, and the',
-              <strong key = 'benefit'> expected outcome/benefit </strong>,
-              'of the research information will be posted publicly on the All of Us website to ' +
-              'inform the research participants.']} required>
-          <TextArea value={this.state.workspace.description}
-                    onChange={v => this.setState(fp.set(['workspace', 'description'], v))}/>
-        </WorkspaceEditSection>
-        <WorkspaceEditSection
-            header='Please select all data use categories that apply for your current study'
-            text='These are for informational purposes only and do not affect or configure your
-            new workspace.'>
-          <WorkspaceCategory style={{width: '100%'}}
-              item={ResearchPurposeItems.diseaseFocusedResearch}
-              value={this.state.workspace.researchPurpose.diseaseFocusedResearch}
-              onChange={v => this.updateWorkspaceCategory('diseaseFocusedResearch', v)}>
-            <TooltipTrigger
-                content='You must select disease focused research to enter a disease of focus'
-                disabled={this.state.workspace.researchPurpose.diseaseFocusedResearch}>
-              <TextInput value={this.state.workspace.researchPurpose.diseaseOfFocus}
-                         style={{
-                           width: 'calc(50% - 2rem)',
-                           border: '1px solid #9a9a9',
-                           borderRadius: '5px'
-                         }}
-                         placeholder='Name of Disease' onChange={v =>
-                  this.setState(fp.set(['workspace', 'researchPurpose', 'diseaseOfFocus'], v))}
-                         disabled={!this.state.workspace.researchPurpose.diseaseFocusedResearch}/>
-            </TooltipTrigger>
-          </WorkspaceCategory>
-          <div style={{display: 'inline-block'}}>
-            <div style={{display: 'flex'}}>
+            tooltip={toolTipText.billingAccount}/>
+        <WorkspaceEditSection header='Research Use Statement Questions'
+            text={['The AoU Research Program requires each user of AoU data to provide a \
```

Comment:
I know this is an existing problem, but can we make this prop more meaningful than `text`? I'd expect `text` to be a child. Maybe `sectionDescription`?